cluster/addons/istio/auth/istio-auth.yaml

```diff
@@ -8,6 +8,7 @@ kind: Namespace
 metadata:
   name: istio-system
   labels:
+    istio-injection: enabled
```

Comment:
This should not be applied to istio-system namespace. It should be applied to the "default" namespace.

Reply:
Done.